src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -37,6 +39,7 @@ link_directories(${SHARED_MODULES}/indexer_connector/build)
 
 add_subdirectory("src/scanOrchestrator")
 add_subdirectory("src/databaseFeedManager")
+add_subdirectory(tests)
```

Comment:
I think tests subdirectory should only be added when compilation define "UNIT_TEST" is set.
In my similar PR use this piece of code:
```
if(UNIT_TEST)
  target_link_libraries(vulnerability_scanner scan_orchestrator database_feed gcc_s)
  if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
    target_link_libraries(vulnerability_scanner -fprofile-arcs)
  else()
    target_link_libraries(vulnerability_scanner gcov)
  endif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")

  add_subdirectory(tests)
else()
  target_link_libraries(vulnerability_scanner scan_orchestrator database_feed router content_manager indexer_connector gcc_s)
  if(FSANITIZE)
    set(CMAKE_CXX_FLAGS_DEBUG "-g -fsanitize=address,leak,undefined")
  endif(FSANITIZE)
  #add_subdirectory(testtool)
endif(UNIT_TEST)
```

Reply:
You're right as we discussed earlier.